pkg/chrootarchive/archive_linux.go

```diff
@@ -0,0 +1,14 @@
+// +build !windows
```

Comment:
This file should be named `archive_unix.go` and keep `!windows`. `_linux` would make it build only on Linux.